ui/app/AppLayouts/Profile/controls/ShowcaseDelegate.qml

```diff
@@ -84,35 +93,31 @@ StatusDraggableListItem {
                     id: menuLoader
                     active: false
                     sourceComponent: StatusMenu {
+                        width: Math.max(everyoneAction.implicitWidth, contactsAction.implicitWidth, idVerifiedContactsAction.implicitWidth) + margins * 2
                         onClosed: menuLoader.active = false
                         StatusMenuHeadline { text: qsTr("Show to") }
 
                         ShowcaseVisibilityAction {
-                            ButtonGroup.group: showcaseVisibilityGroup
+                            id: everyoneAction
                             showcaseVisibility: Constants.ShowcaseVisibility.Everyone
-                            text: qsTr("Everyone")
-                            checked: root.showcaseVisibility === showcaseVisibility
+                            text: qsTr("Everyone%1").arg(!enabled ? qsTr(" (set account to Everyone)") : "")
```

Comment:
```suggestion
                            text: enabled ? qsTr("Everyone") : qsTr("Everyone (set account to Everyone)")
```

Reply:
Please do it like this :) No more guessing what "%1" is and is also less error prone towards the translators who might just insert the space by mistake, thinking it's a typo

Reply:
(same below ofc)